src/main/java/uk/gov/pay/directdebit/mandate/services/MandateService.java

```diff
@@ -90,7 +90,10 @@ public Mandate createMandate(Map<String, String> mandateRequestMap, String accou
                             gatewayAccount,
                             MandateType.valueOf(mandateRequestMap.get("agreement_type")),
                             RandomIdGenerator.newId(),
-                            null,
+                            // TODO:
+                            // when we introduce GoCardless gateway accounts to work with create mandate,
+                            // then modify appropriate mandate reference values
+                            PaymentProvider.SANDBOX.equals(gatewayAccount.getPaymentProvider()) ? "sandbox-default" : "gocardless-default",
```

Comment:
For sandbox, it would be more realistic to generate the mandate reference manually

Fine with using something static for GoCardless because we will be using mandate references generated by GoCardless by the time anyone starts using this